aiida-registry-app/src/Components/Details.jsx

```diff
@@ -0,0 +1,233 @@
+import { Link } from 'react-router-dom';
+import jsonData from '../plugins_metadata.json'
+import base64Icon from '../base64Icon';
+
+import Markdown from 'markdown-to-jsx';
+
+const entrypointtypes = jsonData["entrypointtypes"]
+const plugins  = jsonData["plugins"]
+const status_dict = jsonData["status_dict"]
+const currentPath = import.meta.env.VITE_PR_PREVIEW_PATH || "/aiida-registry/";
+
+
+function Details({pluginKey}) {
+    const value = plugins[pluginKey];
+    window.scrollTo(0, 0);
+    document.documentElement.style.scrollBehavior = 'smooth';
+    return (
+      <>
+
+      <div id="details" className='fade-enter'>
+      <h1 className='plugin-header'>
+          AiiDA plugin package &quot;<a href={value.code_home}>{value.name}</a>&quot;
+      </h1>
+      <Link to={'/'}><p>&lt; back to the registry index</p></Link>
```

Comment:
I notice that when the mouse hover not on the text but on the same line the hyperlink is still activated, this may cause coincidently trigger to go back to the `/`. I guess this is the code related. 

(You can check by moving the mouse not on the "back to the registry index" but on the same horizontal line.)